ipaserver/install/kra.py

```diff
@@ -47,9 +49,9 @@ def install_check(api, replica_config, options):
 
     if replica_config is not None:
         if not api.Command.kra_is_enabled()['result']:
-            raise RuntimeError(
-                "KRA is not installed on the master system. Please use "
-                "'ipa-kra-install' command to install the first instance.")
+            root_logger.info("KRA is not installed on the master system. "
```

Comment:
I also asked for this, it allows debugging to figure out what happened.
Perhaps we should log ad the debug level, not info, but I definitely would like to have an early indication of what the installer thinks is going on.